trillian/migrillian/main.go

```diff
@@ -140,3 +152,33 @@ func newPreorderedLogClient(ctx context.Context, conn *grpc.ClientConn, treeID i
 	pref := fmt.Sprintf("%d", *logID)
 	return core.NewPreorderedLogClient(log, tree, pref)
 }
+
+func getElectionFactory() (factory election.Factory, closeFn func()) {
```

Comment:
(nit) Named returns are [generally avoided](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters), and you don't really need them here.